chrysler_pacifica_ehybrid_s_2019/SubsystemControllerParams.yaml

```diff
@@ -26,7 +26,6 @@ hardware_interface:
       unmanaged_required_nodes: ['']
       excluded_namespace_nodes:
         - /hardware_interface/velodyne_lidar_driver_wrapper
-        - /hardware_interface/carma_novatel_driver_wrapper_node
```

Comment:
Can we also remove the velodyne node from the localization parameters near the bottom of the file?